config/templates/blueprint-manifest.yml.ejs

```diff
@@ -53,7 +54,6 @@ if (!spec.previous_manifest) {
     blueprint: &blueprint
     properties:
       port: 8080
```

Comment:
can you add this line: actionResponse: <%= JSON.stringify(spec.actions) %> This is missed out in your commit